analysis/export/CSVExporter/src/main/kotlin/de/maibornwolff/codecharta/exporter/csv/ParserDialog.kt

```diff
@@ -5,12 +5,14 @@ import com.github.kinquirer.components.promptInput
 import com.github.kinquirer.components.promptInputNumber
 import de.maibornwolff.codecharta.tools.interactiveparser.ParserDialogInterface
 import java.math.BigDecimal
+import java.nio.file.Paths
 
 class ParserDialog {
     companion object : ParserDialogInterface {
         override fun collectParserArgs(): List<String> {
             val inputFileName = KInquirer.promptInput(
                 message = "What is the cc.json file that has to be parsed?",
+                    hint = Paths.get("").toAbsolutePath().toString()
```

Comment:
Is  this misleading because a cc.json is requested instead of a directory path? Or is the directory scanned for cc.json if no cc.json is specified in the path?

Reply:
You are absolutely right, in some other dialogs I added this, but here I forgot!